eng/common/scripts/stress-testing/find-all-stress-packages.ps1

```diff
@@ -62,6 +69,31 @@ function GetUsername() {
     return $stressUser.ToLower()
 }
 
+function ScenariosMatrixGeneration(
+    [string]$matrixFilePath
+) {
+    $yamlConfig = Get-Content $matrixFilePath -Raw
+    $matrix = GenerateMatrix (GetMatrixConfigFromYaml $yamlConfig) "sparse"
```

Comment:
I think this is fine to start, but it would be good to make the entrypoint `Create-JobMatrix.ps1` instead of calling the inner functions, and to make a few updates to that file to allow for the flexibility (e.g. not logging the vso command based on env vars, etc.). I want to support the `-Filter` and `-Replace` parameters in the stress deploy script as well.

Reply:
Do we only want Filter and Replace?